### ISSUE TITLE
Feature: Please add atcoder problem set as well

### ISSUE THREAD

**Issue by shikhar13012001** [Surya1231/ContestMania#21]
I was hoping if there could be an atcoder problem set as well :)

**Comment by Surya1231** [comment]
Thank you so much, Currently I am busy for few days, so will not be able to work on this. Bit it would be great if you can contribute for this one

**Comment by shikhar13012001** [comment]
Yes I would like to contribute to this , it will be fun and learning for me
